packages/opentelemetry-metrics/test/Meter.test.ts

```diff
@@ -578,57 +555,30 @@ describe('Meter', () => {
         );
       });
 
-      it('should not set the instrument data when disabled', async () => {
-        const valueRecorder = meter.createValueRecorder('name', {
-          disabled: true,
-        }) as ValueRecorderMetric;
+      it('should accept negative (and positive) values', async () => {
+        const valueRecorder = meter.createValueRecorder('name');
         const boundValueRecorder = valueRecorder.bind(labels);
-        boundValueRecorder.record(10);
+        boundValueRecorder.record(-10);
+        boundValueRecorder.record(50);
 
         await meter.collect();
         const [record1] = meter.getBatcher().checkPointSet();
         assert.deepStrictEqual(
           record1.aggregator.toPoint().value as Distribution,
           {
-            count: 0,
-            last: 0,
-            max: -Infinity,
-            min: Infinity,
-            sum: 0,
+            count: 2,
+            last: 50,
+            max: 50,
+            min: -10,
+            sum: 40,
           }
         );
+        assert.ok(
+          hrTimeToNanoseconds(record1.aggregator.toPoint().timestamp) >
+            hrTimeToNanoseconds(performanceTimeOrigin)
+        );
       });
 
-      it(
-        'should accept negative (and positive) values when absolute is set' +
```

Comment:
This test case was modified to not pass any value for 'absolute' and then use existing assertions to assert both positive and negative values are accepted.